### ISSUE TITLE
Task and Subtasks linked to the master task

### ISSUE THREAD

**Issue by rarvin** [kevinpapst/kimai-plugins#14]
In a task i define the to date. And for one task I define multiiple subtasks. The end data for all of the subtasks is the to date of the master or main task. All sub tasks can run in parallel, but cant be done by the same user hence ideally each sub task would be an independent task but one that has a linking to the main task and all time against all of the sub-tasks get accumulated to the main task. 

Pausing of the main task puts on hold all of the subtasks, but pausing of one of the subtask does not put the main one on hold. 

It would be like just referencing 
main
       child 1
       child 2
       child 3
       child 4

or even more powerful

master 
         child1
               child1.1
               child1.2
        child2
        child3
               child3.1
               child3.2
                          child3.2.1
       child4

These may be difficult but linking one record to a master and so on is what makes this a chain or tasks. A task creation where you can + add master then if you need add child and keep going. Finally you have all these pending tasks which ideally are individual tasks but can be traced back using the linking.
